src/averageAges.js

```diff
@@ -15,7 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
+  let men = people.filter(person => person.sex === 'm');
+
+  men = century
+    ? men.filter(man => Math.ceil(man.died / 100) === century)
+    : men;
+
+  const ages = men.map(person => person.died - person.born);
+  const sumOfAges = ages.reduce((sum, age) => sum + age, 0);
```

Comment:
calling map method is redundant here
you can go throw initial array and calculate born - died value inside the callback function in reduce method